src/RoleObjects/AccessibilityObject.js

```diff
@@ -151,23 +150,141 @@ export default class AccessibilityObject {
   }
 
   /**
-   * Sets the keyboard shortcut to click the DisplayObject
+   * The translate attribute specifies whether the content of an element should be translated or not.
    * @access public
    * @param {String} keycode - string containing the letter to use for the keyboard shortcut
    */
   set accessKey(keycode) {
-   this._reactProps.accessKey = keycode;
+    this._reactProps.accessKey = keycode;
   }
 
   /**
-   * Retrieves the keyboard shortcut to click the DisplayObject
+   * The translate attribute specifies whether the content of an element should be translated or not.
    * @access public
    * @returns {String} string containing the letter to use for the keyboard shortcut
    */
   get accessKey() {
     return this._reactProps.accessKey;
   }
 
+
+  /**
+   * Sets the keyboard shortcut to click the DisplayObject
+   * @access public
+   * @param {String} str - "yes" or "no" Specifies that the content of the element should be translated or not
+   */
+  set translate(str) {
+    this._reactProps.translate = str;
+  }
+
+  /**
+   * Retrieves the keyboard shortcut to click the DisplayObject
+   * @access public
+   * @returns {String} string containing the letter to use for the keyboard shortcut
+   */
+  get translate() {
+    return this._reactProps.translate;
+  }
+
+
+  /**
+   * The spellcheck attribute specifies whether the element is to have its spelling and grammar checked or not.
+   * @access public
+   * @param { boolean } bool
+   */
+  set spellcheck(bool) {
+    this._reactProps.spellCheck = bool;
+  }
+
+
+  /**
+   * The spellcheck attribute specifies whether the element is to have its spelling and grammar checked or not.
+   * @access public
+   * @returns {boolean} boolean: true or false
+   */
+  get spellcheck() {
+    return this._reactProps.spellCheck;
+  }
+
+
+  /**
+   * The contenteditable attribute specifies whether the content of an element is editable or not.
+   * @access public
+   * @param { boolean } bool true or false
+   */
+  set contenteditable(bool) {
+    this._reactProps.contentEditable = bool;
+  }
+
+  /**
+   * The contenteditable attribute specifies whether the content of an element is editable or not.
+   * @access public
+   * @returns {boolean} true or false
+   */
+  get contenteditable() {
+    return this._reactProps.contentEditable;
+  }
+
+
+  /**
+   * Sets the title for the DisplayObject
+   * @access public
+   * @param {String} title - string mouse hover show text of title as tooltip
```

Comment:
While it is most often shown this way, that presentation is not guaranteed and since the DOM lives under the canvas, browsers shouldn't do that in this case, but ATs might use the string.  I suggest a different description along the lines of "string to convey extra information about the DisplayObject"